### DISCUSSION TITLE
Page/Node wont load properly

### DISCUSSION THREAD

**Discussion by Tommythec** [Chia-Network/chia-blockchain#5280] [Support]
Hi, Having completed the connection process, the page will now not load and just looks like this (see pic) this has been going on for days now, is there anything i need to do?
![Screenshot 2021-05-17 at 12 29 19](https://user-images.githubusercontent.com/84323194/118488264-79c2c000-b713-11eb-9a82-13ccbf0770c7.png)

thanks
Tom

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
